pyproject.toml

```diff
@@ -160,22 +160,13 @@ PKG_CONFIG_PATH="/project/.openblas"
 LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/project/.openblas/lib"
 
 [tool.cibuildwheel.macos]
-# universal2 wheels are not supported (see gh-21233), use `delocate-fuse` if you need them
-archs = "x86_64 arm64"
```

Comment:
The above two lines should be kept. Removing them caused 10 jobs to be added for `universal2`, which we don't want.

Reply:
I just deleted the test skip line below; that one didn't not seem useful since the job didn't exist due to the above two lines to begin with.

Reply:
Can you point to where the universal wheels are being built? The `CIBW_BUILD` variable controls what is built, e.g. `cp312-macosx_arm64` shouldn't make a universal wheel.

Reply:
Anyway, will add `*_universal2` to the list of wheels to be skipped during building.

Reply:
I'm sorry, it looks like I was hallucinating. I distinctly saw a `universal2` tag, but cannot find it back - except for in the dependency installs, where `ninja` is installed from a universal2 wheel on arm64. 

Still, can we keep the comment line? Saves archeology the next time this comes up.

Reply:
I put the comment back already.

Reply:
Thanks - resolving this comment then. Sorry again for the noise.